llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -24356,3 +24356,8 @@ RISCVTargetLowering::emitDynamicProbedAlloc(MachineInstr &MI,
   MF.getInfo<RISCVMachineFunctionInfo>()->setDynamicAllocation();
   return ExitMBB->begin()->getParent();
 }
+
+ArrayRef<MCPhysReg> RISCVTargetLowering::getRoundingControlRegisters() const {
```

Comment:
I think we should check `Subtarget->hasStdExtFOrZfinx()` before returning `{RISCV::FRM, RISCV::FFLAGS}`. If that is false we should return an emtpy set.